mapadroid/websocket/WebsocketServer.py

```diff
@@ -273,7 +273,7 @@ def __internal_worker_join(self):
             return None
 
         if not self.__data_manager.is_device_active(origin):
-            logger.warning('Origin %s is currently paused.  Unpause through MADmin to begin working', origin)
+            logger.warning('Origin {} is currently paused.  Unpause through MADmin to begin working', origin)
```

Comment:
Maybe you can also remove the double space?!

Reply:
Done.